src/doc/src/reference/semver.md

```diff
@@ -1210,10 +1210,13 @@ Mitigating strategies:
     * Understand you may need to deal with resolving new warnings whenever you update your dependencies.
     * Place `deny(warnings)` behind a [feature][Cargo features], for example `#![cfg_attr(feature = "deny-warnings", deny(warnings))]`.
       Set up your automated CI to check your crate with the feature enabled, possibly as an allowed failure with a notification.
+      Beware that features are exposed to users, so you may want to clearly document it as something that is not to be used.
```

Comment:
Documentation is insufficient since its buried somewhere no one will look.

Unless we are willing to recommend an unstable feature convention (clap uses `unstable-` prefixes), I'd rather we just not list this alternative due to the pitfalls and the thoroughness needed to document it.